jenkins/docker/docker-build.jenkinsfile

```diff
@@ -77,9 +82,8 @@ pipeline {
                 script {
                     echo 'The docker-build workflow will only push docker images to staging, please use docker-copy to move the image to other repositories'
                     checkout([$class: 'GitSCM', branches: [[name: "${DOCKER_BUILD_GIT_REPOSITORY_REFERENCE}" ]], userRemoteConfigs: [[url: "${DOCKER_BUILD_GIT_REPOSITORY}" ]]])
-                    def CREDENTIAL_ID = "jenkins-staging-dockerhub-credential"
-                    sh("echo Account: ${CREDENTIAL_ID}")
-                    withCredentials([usernamePassword(credentialsId: CREDENTIAL_ID, usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD')]) {
+                    echo "Account: dockerhub staging"
```

Comment:
Was this intentional and required?

Reply:
Sorry what do you mean by intentional?
This var was initially added to note which account we are logging into.
And the CREDENTIAL_ID is not needed anymore after moving to 1password.

Reply:
I meant do we need to add `echo` ?

Reply:
We dont have to add echo just to inform it is staging.